test/integration.js

```diff
@@ -258,6 +258,38 @@ tape('POST /channel/{id}/{validator-messages}: wrong signature', function(t) {
 	.catch(err => t.fail(err))
 })
 
+tape('POST /channel/{id}/{validator-messages}: wrong root hash', function(t) {
+	const stateRoot = "6def5a300acb6fcaa0dab3a41e9d6457b5147a641e641380f8cc4bf5308b16f1"
+
+	fetch(`${followerUrl}/channel/${dummyVals.channel.id}/validator-messages`, {
```

Comment:
did you try if this test fails w/o the check?

Reply:
Yes, it passes without the check and fails with the check

Reply:
yes I did, it passes without the check and fails with the check